data/main/topics.json

```diff
@@ -5,7 +5,7 @@
       {
         "description": "A software project may come in the form of:",
         "hint": "Recall the types of software projects that exist.",
-        "explanation": "A brown-field project involves developing a product to replace/update an existing software product\nwhile a green-field project involves developing a totally new system with no precedent",
+        "explanation": "",
```

Comment:
Hey I think you accidentally removed some explanations here and some other places

Reply:
My apologies, I've added them back